### DISCUSSION TITLE
error Grid.Background Imagebrush

### DISCUSSION THREAD

**Discussion by wosiyourfather** [AvaloniaUI/Avalonia#12287] [Q&A] [answered]
<Grid Grid.Row="2" >
			<Grid.Background>
				<ImageBrush ImageSource="/Assets/底部.png"/>
			</Grid.Background>
![微信图片_20230722092549](https://github.com/AvaloniaUI/Avalonia/assets/53590491/1c4234a5-4662-4101-b4e2-ca1e373f0d4f)

**Comment by wosiyourfather** [comment]
```xaml
<Grid.Background>
				<ImageBrush ImageSource="/Assets/底部.png"/>
			</Grid.Background>
```
why error Unable to resolve suitable regular or attached property ImageSource on type Avalonia.Base:Avalonia.Media.ImageBrush 第 116 行，位置 6。	SCADA	D:\huafeng\SCADA\SCADA\SCADA\Views\FRAME.axaml	116

**Reply by maxkatz6** [reply, marked as answer]
`ImageBrush` doesn't have `ImageSource` property...
It has `Source`.
